eslint.config.ts

```diff
@@ -0,0 +1,55 @@
+import eslint from "@eslint/js";
+import solid from "eslint-plugin-solid";
+import * as globals from "globals";
+import tseslint from "typescript-eslint";
+import tsParser from "@typescript-eslint/parser";
+
+export default [
+	{
+		name: "global-ignores",
+		ignores: [
+			"**/dist/**",
+			"**/node_modules/**",
+			".github/",
+			".solid/",
```

Comment:
```suggestion
			".solid/",
			".vinxi/",
			".netlify/",
```

When I checked out this branch, I got many errors from these two directories. You can also enable [`.gitignore` support](https://eslint.org/docs/latest/use/configure/ignore#including-gitignore-files) if you want. I thought Eslint 9 respects `.gitignore` but turned out it needs to be added manually.